egs/wsj/s5/utils/parallel/pbs.pl

```diff
@@ -52,16 +52,12 @@
 # command qsub -v PATH -S /bin/bash -l arch=*64*
 # option mem=* -l mem_free=$0,ram_free=$0
 # option mem=0          # Do not add anything to qsub_opts
-# option num_threads=* -pe smp $0
```

Comment:
It's not safe to remove the num-threads option, many scripts use it.  PBS pro must have a mechanism to request more than one core per job.